css/style.css

```diff
@@ -0,0 +1,238 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+@import url("https://fonts.googleapis.com/css?family=Source+Sans+Pro:400|Oswald:400");
+
+html {
+  font-size: 100%;
+} /*16px*/
```

Comment:
Lets try to avoid leaving unused code in our code base as it makes our code look untidy